### PR TITLE
Fixes SEO review

### PR DESCRIPTION
**Pull request by Adebesin-Cell**
Fixes SEO review

fixes https://github.com/EveripediaNetwork/issues/issues/784

### CODE REVIEW

**Comment by kesar** [comment]
done:
no index for revisions
revisions title should be same as wiki (missing category i think)

are you sure all pages have proper titles / descriptions ?
review seo titles / descriptions per page
review other static pages / sections to make sure they have title and description

**Review by Softdev1 — APPROVED** [review]

**Review by kesar — CHANGES_REQUESTED** [review]
done:
no index for revisions
revisions title should be same as wiki (missing category i think)

are you sure all pages have proper titles / descriptions ?
review seo titles / descriptions per page
review other static pages / sections to make sure they have title and description

File: src/components/SEO/Activity.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import { NextSeo } from 'next-seo'
+
+const ActivityHeader = () => (
+  <NextSeo
+    title="Activities | IQ.Wiki | Largest Blockchain & Crypto Encyclopedia "
```
Comment (kesar) [inline src/components/SEO/Activity.tsx:6]: let me review all this before merging 👍🏻

Comment (Royal-lobster) [reply src/components/SEO/Activity.tsx:6]: @Adebesin-Cell you can remove the`| IQ.Wiki | Largest Blockchain & Crypto Encyclopedia` currently we have the default tralier as `| IQ.Wiki |` or you can also edit the template if you are aiming for the short description in title

https://github.com/EveripediaNetwork/ep-ui/blob/51217485ade526e56bd6f4ddfa6f41e9b03b6454/src/components/SEO/Default.tsx#L12

File: src/components/SEO/Default.tsx [hunk 2]
```diff
@@ -9,7 +9,7 @@ interface SEOHeaderProps {
 const SEOHeader = ({ router }: SEOHeaderProps) => (
   <DefaultSeo
     title="IQ.Wiki | Largest Blockchain & Crypto Encyclopedia"
-    titleTemplate="%s | IQ.Wiki"
+    titleTemplate="%s | IQ.Wiki | Largest Blockchain & Crypto Encyclopedia"
```
Comment (kesar) [inline src/components/SEO/Default.tsx:12]:
```suggestion
    titleTemplate="%s | IQ.Wiki"
```

File: src/components/SEO/Static.tsx [hunk 3]
```diff
@@ -89,13 +88,12 @@ export const PrivacyPolicySEO = () => (
 
 export const CareersHeader = () => (
   <NextSeo
-    title="Jobs | Opportunities at IQ.Wiki | Largest Blockchain & Crypto Encyclopedia "
+    title="Jobs | Opportunities at"
```
Comment (kesar) [inline src/components/SEO/Static.tsx:91]:
```suggestion
    title="Jobs"
```

File: src/components/SEO/Static.tsx [hunk 4]
```diff
@@ -89,13 +88,12 @@ export const PrivacyPolicySEO = () => (
...
     description="Join us to spread the knowledge of blockchain | Search Job Openings across Everipedia Network"
     titleTemplate="%s | IQ.Wiki"
     canonical="https://iq.wiki/static/careers"
     openGraph={{
-      title:
-        'Jobs | Opportunities at IQ.Wiki | Largest Blockchain & Crypto Encyclopedia ',
+      title: 'Jobs | Opportunities at',
```
Comment (kesar) [inline src/components/SEO/Static.tsx:96]:
```suggestion
      title: 'Jobs',
```

File: src/pages/search/[query].tsx [hunk 5]
```diff
@@ -17,10 +17,11 @@ import ActivityCard from '@/components/Activity/ActivityCard'
 import { WikiPreview } from '@/types/Wiki'
 import { Link } from '@/components/Elements'
 
-const SearchQuery = () => {
-  const { query: queryParam } = useRouter()
-  const query = queryParam.query as string
+interface SearchQueryProps {
```
Comment (kesar) [inline src/pages/search/[query].tsx:20]: im seeing unrelated to this branch changes, can you make sure this is not gonna lead in some overwrite / unexpected bugs ?

Comment (Adebesin-Cell) [reply src/pages/search/[query].tsx:20]: The search result page was acting weird during my SEO review yesterday. So, I had to fix it to give proper titles and descriptions. I also improved the loading state. And I resolved some conflicts on the branch yesterday.

**Review by kesar — APPROVED** [review]

**Review by Royal-lobster — APPROVED** [review]

**Pull request merged**